src/OpenTelemetry.Instrumentation.Dependencies/Implementation/SqlClientDiagnosticListener.cs

```diff
@@ -91,11 +90,10 @@ public override void OnCustom(string name, Activity activity, object payload)
 
                             if (this.commandTypeFetcher.Fetch(command) is CommandType commandType)
                             {
-                                activity.AddTag(DatabaseStatementTypeSpanAttributeKey, commandType.ToString());
```

Comment:
This change was made to avoid the `.ToString` perf hit. Unrelated to the main changes, sorry.

Reply:
:D I read this comment late, and was still trying to figure out this part !